crates/db_schema/src/source/local_user.rs

```diff
@@ -77,7 +77,7 @@ pub struct LocalUserInsertForm {
   #[builder(!default)]
   pub person_id: PersonId,
   #[builder(!default)]
```

Comment:
Could get rid of this and simplify the test code. But most of that is already cleaned up with a helper method on main branch.